operator-common/src/main/java/io/strimzi/operator/common/DefaultAdminClientProvider.java

```diff
@@ -44,60 +43,45 @@
     @Override
     public Admin createAdminClient(String bootstrapHostnames, Secret clusterCaCertSecret, Secret keyCertSecret, String keyCertName) {
         Admin ac;
-        String trustStorePassword = null;
-        File truststoreFile = null;
+        String sslTruststoreCertificates = null;
+        String sslKeystoreKey = null;
+        String sslKeystoreCertificateChain = null;
```

Comment:
The keystore / truststore names usually refer some special certificate store => e.g. JKS or PKCS12 store. What you are using here isn't really a certificate store. So maybe you should name the vars accordingly.